index.bs

```diff
@@ -370,6 +370,18 @@ To <dfn>parse attribution data</dfn> given a [=string=] |input| modulo an intege
 1. Let |clampedDecodedInput| be the remainder when dividing |decodedInput| by |maxData|.
 1. Return |clampedDecodedInput|.
 
+<h3 algorithm id="noising-data-fields">Noising data fields</h3>
+
+This section defines how to noise [=attribution trigger/trigger data=] and
+[=attribution trigger/event source trigger data=].
+
+To <dfn>noise attribution data</dfn> given an integer |input| between 0 (inclusive) and |maxData|
+(exclusive) and a probability |noiseRate|:
+
+1. Let |shouldNoise| be a randomly selected boolean value such that true is selected with probability |noiseRate|.
```

Comment:
Is there any precedence for going the probability approach rather than e.g. a double and comparing against a random double between 0 and 1?

The double approach seems like it leaves less to the imagination